src/modules/dynamic-ui-wrapper/dynamic-ui-wrapper.ts

```diff
@@ -120,6 +137,32 @@ export class DynamicUiWrapper {
     this.currentUserTask = undefined;
   }
 
+  private _finishManualTask(): void {
+    const hasNoCurrentManualTask: boolean = this.currentManualTask === undefined;
```

Comment:
Those prefixes like `has` or `is` are not the ideal way here, try:

```suggestion
    const noManualTaskKnown: boolean = this.currentManualTask === undefined;
```

Or someting like this, then (...)